test/verify/check-docker

```diff
@@ -195,10 +203,58 @@ class TestDocker(MachineCase):
 
         # Delete image itself
         b.click("#image-details-delete")
-        self.confirm()
+        self.deleteImageConfirm()
         b.wait_visible("#containers")
         b.wait_not_in_text('#containers-images', 'busybox:latest')
 
+    def testDeleteImages(self):
+        b = self.browser
+        m = self.machine
+        m.execute("systemctl start docker || systemctl start docker-latest")
+        self.login_and_go("/docker")
+        b.wait_in_text("#containers-images", "busybox:latest")
+
+        # Deleting an image without depending containers has no container list
+        b.wait_visible('#containers-images tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.click('#containers-images tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.click('#containers-images tr:contains("busybox:latest") + tr .btn-delete')
```

Comment:
It feels like this needs to wait for the previous "toggle" click to take effect, i. e. wait for the btn-delete to become visible? (It may be  immediate due to how jQuery works - let's see how it fares on CI)